### PR TITLE
Use RawSQL instead of Create to be able to upgrade to gorm new version

### PR DESCRIPTION
**Pull request by magellancl**
#217 
 
 Making a PR to support the latest gorm upgrade because the gorm fix doesn't seem to be coming anytime soon. 
 
 I did this quickly, but I am a bit wary of potential SQL injection, any review is welcome.

### CODE REVIEW

File: gormigrate.go [hunk 1]
```diff
@@ -458,8 +458,7 @@ func (g *Gormigrate) unknownMigrationsHaveHappened() (bool, error) {
 }
 
 func (g *Gormigrate) insertMigration(id string) error {
-	record := map[string]interface{}{g.options.IDColumnName: id}
-	return g.tx.Table(g.options.TableName).Create(record).Error
+	return g.tx.Exec("INSERT INTO " + g.options.TableName + " (" + g.options.IDColumnName + ") VALUES ('" + id + "')").Error
```
Comment (oxyno-zeta) [inline gormigrate.go:461]: This can lead to SQL Injection and prevent id to have single quote inside. Am I missing something ?

Comment (magellancl) [reply gormigrate.go:461]: It's not much of a PR, more like the draft of a possible fix to get some feedback

Comment (oxyno-zeta) [reply gormigrate.go:461]: Oh I see ! No problem, I was just looking as I saw your PR just after I did one ahah.

**Comment by avakarev** [comment]
@magellancl Thank you for your time and contribution.

**Pull request closed without merging**